test/unit/babel_client_test.js

```diff
@@ -562,23 +562,49 @@ describe("Babel Node Client Test Suite", function(){
             });
 
             var createAnnotation = function(){
-                return babelClient.createAnnotation('token', {hasBody:{format:'text/plain', 'type':'Text'}, annotatedBy:'Gordon Freeman'}, function(err, result){});
+                return babelClient.createAnnotation('token', {hasBody:{format:'text/plain', 'type':'Text'}, annotatedBy:'Gordon Freeman'}, {}, function(err, result){});
             };
 
             createAnnotation.should.throw("Missing data: hasTarget");
             done();
         });
-        it("- create annotation should return an error if hasTarget.uri not supplied", function(done){
+        it("- create annotation should return an error if hasTarget as single object has no uri supplied", function(done){
             var babelClient = babel.createClient({
                 babel_host:"http://babel",
                 babel_port:3000
             });
 
             var createAnnotation = function(){
-                return babelClient.createAnnotation('token', {hasBody:{format:'text/plain', 'type':'Text'}, hasTarget:{}, annotatedBy:'Gordon Freeman'}, function(err, result){});
+                return babelClient.createAnnotation('token', {hasBody:{format:'text/plain', 'type':'Text'}, hasTarget:{}, annotatedBy:'Gordon Freeman'}, {}, function(err, result){});
             };
 
-            createAnnotation.should.throw("Missing data: hasTarget.uri");
+            createAnnotation.should.throw("Missing data: hasTarget.uri is required");
+            done();
+        });
+        it("- create annotation should return an error if hasTarget as array contains one or more objects with no uri", function(done){
+            var babelClient = babel.createClient({
+                babel_host:"http://babel",
+                babel_port:3000
+            });
+
+            var createAnnotation = function(){
+                return babelClient.createAnnotation('token', {hasBody:{format:'text/plain', 'type':'Text'}, hasTarget:[{uri: 'foo'}, {}], annotatedBy:'Gordon Freeman'}, {}, function(err, result){});
+            };
+
+            createAnnotation.should.throw("Missing data: hasTarget.uri is required");
+            done();
+        });
+        it("- create annotation should return an error if hasTarget contains unrecognised property", function(done){
```

Comment:
ive added another test for when this is an array, in this case hasTarget is a single object